source/extensions/common/tap/extension_config_base.cc

```diff
@@ -16,14 +16,13 @@ ExtensionConfigBase::ExtensionConfigBase(
     : proto_config_(proto_config), config_factory_(std::move(config_factory)), tls_slot_(tls) {
   tls_slot_.set([](Event::Dispatcher&) { return std::make_shared<TlsFilterConfig>(); });
 
-  switch (proto_config_.config_type_case()) {
+  switch (proto_config_.config_type_case())
```

Comment:
Seems like a mistake (the opening brackets should be kept, and the closing one in line 26 should match the opening one in line 20)